package.json

```diff
@@ -77,7 +77,7 @@
     "eth-gas-reporter/colors": "1.4.0"
   },
   "dependencies": {
-    "@mean-finance/dca-v2-core": "^2.1.0",
+    "@chainlink/contracts": "0.4.2",
```

Comment:
In core, we are using `"@chainlink/contracts": "0.2.2"`, are we aware of the major changes and what do they imply?

Reply:
That's a good question! We are only using constants from this repository, and I checked and they haven't changed. But thanks for bringing it up!